.circleci/workflows.yml

```diff
@@ -298,6 +305,50 @@ jobs:
           name: Unit tests
           command: yarn test:unit
 
+  build-and-release-deployment-artifacts-and-abis-to-npm:
+    executor: vm
+    steps:
+      - initialize-environment
+      - node/install-packages:
+          pkg-manager: yarn
+          app-dir: ~/beamer/deployments
+      - run:
+          name: Log in to npm
+          command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+      - when:
+          condition:
+            and:
+              - equal: [ main, <<pipeline.git.branch>> ]
+              - equal: [ true, <<pipeline.parameters.deployment-package-json-updated>> ]
+          steps:
+            - run:
+                name: Build
+                command: make npm-package
+                working_directory: ~/beamer
+            - run:
+                name: Publish to npm
+                command: yarn publish dist/beamer-bridge-deployments.tgz
+                working_directory: ~/beamer/deployments
+      - when:
+          condition:
+            and: 
+              - not:
+                  equal: [ main, <<pipeline.git.branch>> ]
+              - equal: [ true, <<pipeline.parameters.deployment-artifacts-updated>> ]
```

Comment:
Nitpick: you can just say
```
- <<pipeline.parameters.deployment-artifacts-updated>>
```

Reply:
I just started changing it, but I don't think this makes it better:
![image](https://github.com/beamer-bridge/beamer/assets/693770/7b561e57-e97c-44be-9519-670da0ee987f)

At least for my brain this is confusing at first. I have to now convert `-` to mean `equal` and then think about what is 
`pipeline.parameters.deployment-artifacts-updated` supposed to be equal to?

Reply:
Feel free to leave as-is, I just wanted us to be consistent. It's also fine as-is. :+1: